portfolio/src/main/webapp/placesGallery.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Favourite Places Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Favourite Places Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images/place1.jpg"><img src="images/place1.jpg"/></a>
+        <a href="images/place2.jpg"><img src="images/place2.jpg"/></a>
+        <a href="images/place3.jpg"><img src="images/place3.jpg"/></a>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
nit: newline EoF :)

There are tools can help you to take care of these tedious consistency check, they are called linters.
For example, you can take a look of https://prettier.io/. This is pretty popular in the JavaScript Community.

Reply:
Thank you, this is very helpful!